elements/additionalpower.lua

```diff
@@ -1,31 +1,32 @@
 --[[
 # Element: Additional Power Bar
 
-Handles updating and visibility of a status bar displaying the player's additional power, such as Mana for Balance
+Handles updating and visibility of a status bar that displays player's additional power, such as Mana for Balance
 druids.
 
 ## Widget
 
-AdditionalPower - A StatusBar to represent the player's current additional power.
+AdditionalPower - A StatusBar that is used to represent player's additional power.
 
 ## Sub-Widgets
 
-.bg - A Texture, which functions as a background. It will inherit the color of the main StatusBar.
+.bg - A Texture that is used as a background. Inherits the widget's color.
 
 ## Notes
 
-The default StatusBar texture will be applied if the UI widget doesn't have a status bar texture or color defined.
+Default status bar texture will be applied if the widget doesn't have a texture or a color set.
```

Comment:
"The default status bar texture will be applied if the element doesn't have a texture or a color set."